drivers/power_domain/power_domain_intel_adsp.c

```diff
@@ -26,9 +21,11 @@ static int pd_intel_adsp_set_power_enable(struct pg_registers *reg, bool power_e
 	uint32_t SPA_bit_mask = BIT(reg->SPA_bit);
 
 	if (power_enable) {
-		sys_write32(sys_read32(reg->wr_address) | SPA_bit_mask, reg->wr_address);
+		sys_write32(sys_read32(ACE_DfPMCCU->dfpwrctl) | SPA_bit_mask,
```

Comment:
@kfrydryx is this driver designed only for ACE?

Reply:
Currently yes. I'm not sure whether there are any plans of using that with CAVS.